slack/views.py

```diff
@@ -41,7 +41,7 @@ def parseacidclean(instructions,user,fn):
             cropped_instructions = cropped_instructions[0:temp.start()].strip()
             temp = temp.groups()
             try:
-                temp = float(temp[0])
+                temp = float(temp[1])
```

Comment:
so just to confirm, this is because the output of groups is: [full match, first group, second group]?

Reply:
the output here of temp.groups() is ('at ', '345', None), so we want to keep the temp only

Reply:
(note that the temp_re bit in line 12 changed in the last MR)